Geekbot.net/Modules/Choose.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Threading.Tasks;
+using Discord.Commands;
+using Geekbot.net.Lib.IClients;
+
+namespace Geekbot.net.Modules
+{
+    public class Choose : ModuleBase
+    {
+        private readonly IRandomClient rnd;
+        public Choose(IRandomClient randomClient)
+        {
+            rnd = randomClient;
+        }
+
+        [Command("choose", RunMode = RunMode.Async), Summary("Let the bot make a choice for you.")]
+        public async Task Command([Remainder, Summary("The choices, sepperated by a ;")] string choices)
+        {
+            var choicesArray = choices.Split(';');
```

Comment:
That is what most bots do, also, `stuff; more stuff` is easier to program and write than `"stuff" "more stuff"`